pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[build-system]
+requires = ["setuptools>=64", "setuptools-scm"] 
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pytest-dependency"
+description = "Manage dependencies of tests"
+authors = [{name = "Rolf Krahl", email = "rolf@rotkraut.de"}]
+readme = "README.rst"
+requires-python = ">=3.4"
+dependencies = [
+  "pytest",
+]
+dynamic = ["version"]
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "Framework :: Pytest",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: Apache Software License",
+  "Operating System :: OS Independent",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
```

Comment:
I prefer to explicitly list the Python version the respective release supports. It makes it easier when you browse PyPI and to answer questions like "what was the last release that did support Python X.Y?"

And, before you ask: yes before making any release I do test with all officially supported Python versions. I'm not relying on the GitHub workflow alone for that.